### PR TITLE
Remove KVO and just ask the webview how tall it is

### LINKED ISSUE #1441: Crash from navigating from issue to inbox
App crashes after viewing MessageKit #449 and returning to inbox

Debug notes:
- User defaults for 449 is empty
- If I interrupt the view from scrolling to bottom when viewing the issue, it doesn’t crash on return to inbox
- After interrupting the scrolling, extra white space is rendered on GitHawk comments. The issue no longer crashes while white space is present

I think this is related to scrolling or the content size. I can prevent the crash from scrolling to the top of the issue before exiting. This issue only crashes on return to inbox. Viewing it from the repo page works fine. I noticed that entering from the repo page doesn’t scroll to bottom automatically as well. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

### PR DESCRIPTION
**Pull request by rnystrom**
Fixes #1441, should help w/ #1339

### CODE REVIEW

File: Classes/Issues/Comments/Html/IssueCommentHtmlCell.swift [hunk 1]
```diff
@@ -161,20 +155,12 @@ final class IssueCommentHtmlCell: IssueCommentBaseCell, ListBindable, UIWebViewD
     }
 
     func webViewDidFinishLoad(_ webView: UIWebView) {
-        webView.isHidden = false
-    }
-
-    // MARK: KVO
+        guard self.isHidden == false,
+            let contentHeight = webView.stringByEvaluatingJavaScript(from: "document.body.offsetHeight") as NSString?
```
Comment (rnystrom) [inline Classes/Issues/Comments/Html/IssueCommentHtmlCell.swift]: Took some experimenting, but this is the right way to ask

**Comment by Sherlouk** [comment]
Does this still work with HTML5 details?

<details>
<summary>Oooh  Collapsible Content</summary>
here's a gif


</details>

**Comment by rnystrom** [comment]
@Sherlouk good call out, it probably won’t L.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rnystrom** [comment]
Confirmed it doesn’t 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Pull request merged**